### DISCUSSION TITLE
Use Apache environment variables in OIDCPathAuthRequestParams and OIDCPathScope directives

### DISCUSSION THREAD

**Discussion by zaic20** [OpenIDC/mod_auth_openidc#594] [Q&A]
Dear All,

I have a quick question related to the following directives:

- OIDCPathAuthRequestParams
- OIDCPathScope

I want to know if it is possible to use apache environment variable as a value of those directives, for example:

OIDCPathAuthRequestParams "org=%{env:MATCH_ORG}"

What I saw is that the value above has not interpreted but just url-encoded and sent along with the Authorization Request as a query parameter (e.g.   org=%25%7Benv%3AMATCH_ORG%7D)

Thanks so much!!

**Comment by zandbelt** [comment]
no, that is currently not supported; I guess it wouldn't be too hard to add however

**Reply by zandbelt** [reply]
Support for this feature was added in https://github.com/zmartzone/mod_auth_openidc/commit/953a6f133c2086c35a60c77e29aef16ec351b11b available from 2.4.11.2rc2 onwards. 
Note that Apache expressions use `reqenv` instead of `env`, so it would become:
```
OIDCPathAuthRequestParams "org=%{reqenv:MATCH_ORG}"
```
